exec/java-exec/src/main/java/org/apache/drill/exec/store/base/filter/FilterPushDownListener.java

```diff
@@ -126,13 +122,7 @@
      * to leave in the query. Those terms can be the ones passed in, or
      * new terms to handle special needs.
      *
-     * @param groupScan the scan node
-     * @param andTerms a list of the CNF (AND) terms, in which each is given
-     * by the Calcite AND node and the derived RelOp expression.
-     * @param orTerm the DNF (OR) term, if any, that includes the Calcite
-     * node for that term and the set of OR terms. Only provided if the OR
-     * term represents a simple list of values (all OR clauses are on the
-     * same column). The OR term itself is AND'ed with the CNF terms.
+     * @param expr
```

Comment:
Could you describe all newly added empty params? We usually don't add empty param docs
Please refer to the doc of `AndNode` to accomplish that